redwood-snapshot-testing/src/androidMain/kotlin/app/cash/redwood/snapshot/testing/ComposeUiTestWidgetFactory.kt

```diff
@@ -40,6 +44,10 @@ object ComposeUiTestWidgetFactory : TestWidgetFactory<@Composable () -> Unit> {
   override fun color(): ColorWidget<@Composable () -> Unit> = ComposeUiColor()
 
   override fun text(): Text<@Composable () -> Unit> = ComposeUiText()
+
+  override fun column(): SimpleColumn<@Composable () -> Unit> = ComposeUiColumn()
+
+  override fun scrollWrapper(): ScrollWrapper<@Composable () -> Unit> = ComposeUiScrollWrapper()
```

Comment:
Both necessary to make this test work!

Reply:
(I didn’t want to use our FlexColumn because I don’t want these tests to make assumptions that it’ll be our own components enclosing them)